lib/msal-browser/src/cache/IWindowStorage.ts

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+/**
+ * Interface object for window storage. TODO: Add link to custom cache docs.
+ */
+export interface IWindowStorage {
+
+    /**
+     * Get the item from the window storage object matching the given key.
+     * @param key 
+     */
+    getWindowStorageItem(key: string): string;
```

Comment:
I would prefer just `getItem, `getItem`, etc. This is arguably wordy and redundant.

Reply:
+1